### PR TITLE
[WIP][NOT READY] Replace references to AWS

### PR DESCRIPTION
**Pull request by dims**
Change-Id: Ifd896b5a6e4f208ce4600613d50f6d80e798ebe1

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: hack/ci/e2e-conformance.sh [hunk 1]
```diff
@@ -52,6 +52,17 @@ init_image() {
...
+        curl -L https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+        python get-pip.py --user
+        rm -f get-pip.py
+      fi
+      if ! command -v ansible &> /dev/null; then
+        version="2.8.0"
+        python -m pip install --user "ansible==${version}"
+        ln -s $HOME/.local/bin/ansible-playbook "$(go env GOPATH)/bin"
+        ln -s $HOME/.local/bin/ansible "$(go env GOPATH)/bin"
+      fi
```
Comment (detiber) [inline hack/ci/e2e-conformance.sh]: :grimacing:

File: hack/ci/e2e-conformance.sh [hunk 2]
```diff
@@ -63,9 +74,10 @@ init_image() {
           rm packer.zip && \
           mv packer "$(go env GOPATH)/bin"
       fi
-      (cd "$(go env GOPATH)/src/sigs.k8s.io/image-builder/images/capi" && \
-      GCP_PROJECT_ID=$GCP_PROJECT GOOGLE_APPLICATION_CREDENTIALS=$GOOGLE_APPLICATION_CREDENTIALS \
-      make build-gce-default)
+      # FIXME : seems to be blowing up in the CI.
```
Comment (vincepri) [inline hack/ci/e2e-conformance.sh]: Maybe we can have another job that publishes the images in the correct project?

File: hack/ci/e2e-conformance.sh (hunk 1 above)
Comment (dims) [reply hack/ci/e2e-conformance.sh]: just a WIP :)

File: hack/ci/e2e-conformance.sh (hunk 2 above)
Comment (dims) [reply hack/ci/e2e-conformance.sh]: y probably!

**Comment by dims** [comment]
this PR served us well! will open fresh clean PR(s) one for the AWS and one for the CI job changes

/close

**Pull request closed without merging**